anchor-reference-server/src/main/java/org/stellar/anchor/reference/AnchorReferenceServer.java

```diff
@@ -28,7 +28,8 @@ public static void start(int port, String contextPath) {
             .bannerMode(OFF)
             .properties(
                 String.format("server.port=%d", port),
-                String.format("server.contextPath=%s", contextPath));
+                String.format("server.contextPath=%s", contextPath),
+                "spring.liquibase.enabled=false");
```

Comment:
Do we really need this line here? It'd be better to leave it in an existent yaml file, of possible.

Reply:
the unit tests specifically dont seem to pick up the value when set in the yaml